packages/@aws-cdk/aws-dynamodb/README.md

```diff
@@ -134,6 +134,9 @@ const globalTable = new dynamodb.Table(this, 'Table', {
 });
 ```
 
+A maximum of 10 tables with replication can be added to a stack without a limit increase.
```

Comment:
Can you add details on what type of limit increase is needed? Maybe link to the relevant quota documentation?